src/main/java/com/williamfiset/algorithms/datastructures/priorityqueue/BinaryHeapQuickRemovals.java

```diff
@@ -16,12 +16,6 @@
 
 public class BinaryHeapQuickRemovals<T extends Comparable<T>> {
```

Comment:
Ty for also taking the time to edit this file, I actually forgot it existed XD. However, I may be removing it soon and the binary tree removals video with it. I have since discovered a less hacky way to remove elements from a Heap-based priority queue that doesn't require maintaining a map. See: https://github.com/williamfiset/Algorithms/issues/165